src/plugins/explore/public/components/data_table/explore_data_table.scss

```diff
@@ -1,4 +1,9 @@
 .explore-table-container {
   height: 100%;
   mask-image: none;
+
+  /* stylelint-disable-next-line @osd/stylelint/no_modifying_global_selectors */
+  .euiFlexGroup {
+    width: 100%;
+  }
```

Comment:
this is going to target all `.euiFlexGroup` within the explore-table-container. Lets not ignore these stylelint group unless absolute necessary.

you can just add a class to `EuiFlexGroup` and target it like that